assets/sass/app.scss

```diff
@@ -10,3 +10,56 @@
 .page-header {
   margin-bottom: $govuk-gutter;
 }
+
+.expanding-panel {
+  border-top: none;
+  &:last-child {
+    border-bottom: none;
+  }
+
+  header {
+    padding-left: 0;
+    position: relative;
+    cursor: pointer;
+    margin: 0;
+  }
+
+  .content {
+    h2:first-child {
+      margin-top:0;
+    }
+  }
+
+  h3 {
+    margin-bottom:15px;
+  }
+
+}
+
+.govuk-heading-m, .application-confirm legend, .panel-list li {
+  font-size: 16px;
+  font-weight: inherit;
+  @media (min-width: 40.0625em) {
+    .govuk-heading-m, .application-confirm legend, .panel-list li {
+      font-size: inherit;
+      line-height: inherit;
+    }
+  }
+}
+
+h2 {
+  @media (min-width: 40.0625em)  {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 1.25;
+  }
+}
+
+.separator {
+  border-bottom: 1px solid $govuk-border-colour;
+  margin-bottom: 15px;
+}
+
+.add-margin {
+  margin-bottom:20px !important;
+}
```

Comment:
eek

Reply:
can we use multiples of `$govuk-gutter`

Reply:
actually let's remove this - there isn't any `dl dt` in the HTML for this page

Reply:
there are in profile.jsx - I need to move it

Reply:
actaully, this is from 4 days ago ... it is not there anymore anyway